source/grid/tria.cc

```diff
@@ -6366,13 +6368,13 @@ namespace internal
                                    {{X, X}}}};
 
               static constexpr dealii::ndarray<unsigned int, 4, 4>
-                quad_lines_tri{{{{0, 8, 5, X}},
-                                {{1, 2, 6, X}},
-                                {{7, 3, 4, X}},
-                                {{6, 7, 8, X}}}};
+                tri_lines_tri{{{{0, 8, 5, X}},
```

Comment:
In the original code, there were variables `quad_lines_quad`, `quad_lines_tri` and `quad_line_vertices_tri`. I believe the first part of each specifier denotes the mesh object, i.e., the face.

So maybe we should call them `face_lines_quad`, `face_lines_tri`, and `face_line_vertices_tri` instead?

Reply:
That's basically what happens a few lines below. Here, we set up two `static` tables, one for refinement of triangle faces and one for refinement of quad faces. These are the "lines within the triangle children of a triangle" (`tri_lines_tri`) and similarly for quads. Then, in line 6387 (new) below, we reference either one or the other, depending on what kind of cell we have here.

So I think that the variable names actually match the intent.